pom.xml

```diff
@@ -1248,7 +1248,14 @@
                     <artifactId>buildnumber-maven-plugin</artifactId>
                     <version>1.0</version>
                 </plugin>
-
+				<plugin>
```

Comment:
Hi Suma, let me know is there any formatting profile for IDE maybe Intellij/Eclipse or should I default of one?